x-pack/plugins/cases/public/components/all_cases/count.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useEffect } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { CaseStatuses } from '../../../common';
+import { Stats } from '../status';
+import { useGetCasesStatus } from '../../containers/use_get_cases_status';
+
+interface CountProps {
+  refresh?: number;
```

Comment:
Why refresh is a number?

Reply:
This way the parent can trigger an update in the child.  When the number changes, a useEffect runs the re-fetch function

Reply:
This is where the update is coming from: https://github.com/elastic/kibana/pull/97149/files/f3a7c5688262ee1f7d79ecf542e09a9d6e12c2c5#diff-49443f63bd02ccea55f5419c52288cbc0f7d6c7859e79dc5ebb75dcf45ff3b31R110-R122

Reply:
I think we can do better that now, we are smarter that a year ago

Reply:
Speak for yourself 😂

Reply:
Lol, I know we did something similar in resolver (the code may have been removed now), but it was to make sure that if multiple requests are made we could abort an earlier one so we didn't get out of sync data. @kqualters-elastic worked on it I believe.

Reply:
I see. Smart!